### PR TITLE
Map out-of-line animation to inline animation to handle styling seman…

### LINKED ISSUE #927: Out-of-line animations are not included in Specified Style Set Processing
Whereas §10.4.4.2 Specified Style Set Processing includes inline animation elements that are conditionally included child elements of E, there doesn't seem to be any step in the process that merges in the out of line animations referenced by the `animate` attribute.

### PR DESCRIPTION
**Pull request by skynavga**
…tics (#927).

Closes #927.

### CODE REVIEW

**Comment by skynavga** [comment]
@nigelmegitt re: https://github.com/w3c/ttml2/pull/930#pullrequestreview-139995754 I have given some thought to timing on out-of-line (OOL) animations, and I have not come up with a rational model for how this might work; nominally, any timing on an out-of-line animation would be in relation to its hierarchy, e.g., animate/animation/head/tt, which would make that timing be expressed in the document temporal coordinate space; then the question is how to synchronize this with timing on possibly multiple referring elements; my conclusion was that it is better to avoid this problem, at least in TTML2 1e, by not permitting timing on OOL animations;

**Comment by skynavga** [comment]
in other words, if you want timing on an animation that does not correspond with the referring elements timebase, then don't use out-of-line animation

**Comment by nigelmegitt** [comment]
If the semantic seems unclear and we want to leave it to implementations, then we should simply say nothing about it. The newly proposed text that creates a copy of the `animate|set` element as an inline child gives a good opportunity to simply copy across the timing attribute values without prior evaluation. I would simply say nothing about this problem and allow timing on OOL animations, then we have parity with TTML1 for `set` at least, in the sense that TTML2 isn't saying what TTML1 didn't say.

**Comment by skynavga** [comment]
@nigelmegitt ok, I can live with that for TTML 1e; will update in a minute

**Comment by skynavga** [comment]
@nigelmegitt updated

File: spec/ttml2.xml [hunk 1]
```diff
@@ -18630,6 +18630,8 @@ elements in the <loc href="#element-vocab-group-metadata"><code>Metadata.class</
 </tr>
 </tbody>
 </table>
+<p>An <loc href="#terms-out-of-line-animation">out-of-line</loc> <el>set</el> element
```
Comment (nigelmegitt) [inline spec/ttml2.xml]: There used to be a line like this for OOL animate elements too - I think it got dropped by accident?

Comment (skynavga) [reply spec/ttml2.xml]: Nope, it's still there. Just not modified (now).

Comment (skynavga) [reply spec/ttml2.xml]: OTOH, `set` was missing this entirely.

Comment (nigelmegitt) [reply spec/ttml2.xml]: Ah, right, I see, thanks.

**Comment by skynavga** [comment]
Merged early per WG resolution to move forward with CR3 CfC.

**Pull request merged**